### PR TITLE
[luxon] add LocalZone.instance

### PR DESCRIPTION
**Pull request by squirly**
See docs https://moment.github.io/luxon/docs/class/src/zones/localZone.js~LocalZone.html#static-get-instance

### CODE REVIEW

File: types/luxon/index.d.ts [hunk 1]
```diff
@@ -589,4 +589,6 @@ export class FixedOffsetZone extends Zone {
 
 export class InvalidZone extends Zone { }
 
-export class LocalZone extends Zone { }
+export class LocalZone extends Zone {
+    static get instance(): LocalZone;
```
Comment (squirly) [inline types/luxon/index.d.ts:593]: Make compatible with target < ECMAScript 5:
```suggestion
    static readonly instance: LocalZone;
```

**Review by peterblazejewicz — COMMENTED** [review]
@squirly mind adding test, thx!

**Review by andrewbranch — APPROVED** [review]

**Comment by andrewbranch** [comment]
Oops @peterblazejewicz, I didn’t see your review.

**Review by andrewbranch — APPROVED** [review]

**Comment by squirly** [comment]
@peterblazejewicz Test added!

**Review by peterblazejewicz — APPROVED** [review]
@squirly thx! LGTM!

**Comment by squirly** [comment]
Ready to merge

**Pull request merged**